packages/aila/src/features/threatDetection/detectors/lakera/LakeraThreatDetector.ts

```diff
@@ -0,0 +1,226 @@
+import { aiLogger } from "@oakai/logger";
+import { z } from "zod";
+
+import {
+  type ThreatCategory,
+  type ThreatDetectionResult,
+  type ThreatSeverity,
+  AilaThreatDetector,
+} from "../AilaThreatDetector";
+import type { Message, LakeraGuardResponse, BreakdownItem } from "./schema";
+import { lakeraGuardRequestSchema, lakeraGuardResponseSchema } from "./schema";
+
+const log = {
+  info: (message: string, data: Record<string, unknown>) => {
+    console.log(`[LakeraThreatDetector] ${message}`, data);
+  },
+  error: (message: string, data: Record<string, unknown>) => {
+    console.error(`[LakeraThreatDetector] ${message}`, data);
+  },
+};
+
+export class LakeraThreatDetector extends AilaThreatDetector {
+  private readonly apiKey: string;
+  private readonly projectId?: string;
+  private readonly apiUrl = "https://api.lakera.ai/v2/guard";
+
+  constructor() {
+    super();
+    const apiKey = process.env.LAKERA_GUARD_API_KEY;
+    const projectId = process.env.LAKERA_GUARD_PROJECT_ID;
+
+    if (!apiKey)
+      throw new Error("LAKERA_GUARD_API_KEY environment variable not set");
+
+    this.apiKey = apiKey;
+    this.projectId = projectId;
+  }
+
+  protected async authenticate(): Promise<void> {
+    // Authentication is handled via API key in headers
+    return Promise.resolve();
+  }
+
+  private mapSeverity(detectorType: string): ThreatSeverity {
+    // Map Lakera detector types to our severity levels
+    switch (detectorType) {
+      case "jailbreak":
+      case "prompt_injection":
+        return "critical";
+      case "pii":
+      case "sensitive_info":
+        return "high";
+      case "harmful_content":
+        return "high";
+      default:
```

Comment:
We might want to signal to ourselves here that there's a type we're not covering

Reply:
This is a stub really - we have since learned that we get L1-4 classifications from Lakera

Reply:
Gotcha, so you thinking we can remove them, or are they still useful?